rpcs3/Emu/System.cpp

```diff
@@ -300,10 +300,8 @@ void Emulator::Init(bool add_only)
 					sys_log.fatal("Failed to fix save data: %s (%s)", pending, fs::g_tls_error);
 					continue;
 				}
-				else
```

Comment:
Yep, these formatting changes definitely need to go into their own commit at the very least.

Reply:
done